Gemfile

```diff
@@ -2,5 +2,15 @@ source 'https://rubygems.org'
 
 gemspec
 
+def gems(*gems)
```

Comment:
Just curious what is it for?

Reply:
It's some copy-paste from Gemfile of Clockwork. I guess this is some Paweł's shortcut for omitting writing `gem` in each line... :shrug:

Reply:
I would actually remove it. I don't like `gems` anymore.